lib/BloqadeLattices/src/tiles.jl

```diff
@@ -0,0 +1,39 @@
+abstract type Supercell{D} end
+
+struct Parallelepiped{D, T} <: Supercell{D}
+    # abstract matrix dimensions already guaranteed to be 2D
+    bounds_inv::AbstractMatrix{T}
```

Comment:
```suggestion
    bounds::::AbstractMatrix{T}
    bounds_inv::AbstractMatrix{T}
```

We need to keep bounds around as well for other functions.